flow.py

```diff
@@ -66,15 +78,15 @@ def unacknowledged_packets(self):
         pass
 
     @unacknowledged_packets.setter
-    def dest(self, unacknowledged_packets):
+    def unacknowledged_packets(self, unacknowledged_packets):
         pass
 
     @property
     def current_packet(self):
         pass
```

Comment:
so in a lot of places with this property stuff you just pass, but i think you get like return self._current_packet or something like that. also in the setter you can like set self._current_packet = current_packet